app/components/ChartSettings/modules/Metadata.js

```diff
@@ -18,16 +18,28 @@ export default class Metadata extends Component {
     title: '',
     caption: '',
     credit: '',
+    subtitle: false,
   };
 
   componentWillMount() {
     this.setState(this.props.metadata);
+    if ('undefined' !== typeof this.props.metadata.subtitle &&
+      false !== this.props.metadata.subtitle) {
+      this.shouldShowMetadata.subtitle = true;
+    }
   }
 
   componentWillReceiveProps(nextProps) {
     this.setState(nextProps.metadata);
   }
 
+  shouldShowMetadata = {
+    title: true,
+    caption: true,
+    credit: true,
+    subtitle: false,
+  }
```

Comment:
Moved it up, but the `react/sort-comp` ESLint rule was throwing a fit about it. It makes more sense to have it there imo, so I disabled that rule around the block.